helmfile/charts/astrago/templates/inference/service.yaml

```
@@ -0,0 +1,18 @@
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.inference.service.type }}
+  ports:
+    - port: {{ .Values.inference.service.port }}
+      targetPort: {{ .Values.inference.service.port }}
+      protocol: TCP
+      name: http
+  selector: 
+    app.kubernetes.io/name: astrago-backend-inference
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (2 > 0)

(empty-lines)